core/store/orm/config.go

```diff
@@ -150,7 +153,8 @@ func init() {
 		EthGasPriceDefault:               *big.NewInt(5000000000),   // 5 Gwei
 		EthMaxGasPriceWei:                *big.NewInt(500000000000), // 500 Gwei
 		EthMinGasPriceWei:                *big.NewInt(1000000000),   // 1 Gwei
-		EthFinalityDepth:                 50,                        // Keeping this >> 11 because it's not expensive and gives us a safety margin
+		EthGasLimitDefault:               500000,
```

Comment:
Are we thinking of changing this for Fantom? Since, it seems to be the same for all chains, I'm wondering does it actually need to be in here?